validation/validate_dc2_run2.1i_object_table.ipynb

```diff
@@ -0,0 +1,713 @@
+    "    plt.colorbar(shrink=0.5, label='objects / bin')\n",
+    "\n",
+    "    # This region isn't quite a polygon.  The sides should be curved.\n",
+    "    wfd_region = Polygon(dc2_run2x_wfd, color='red', fill=False)\n",
+    "    ddf_region = Polygon(dc2_run2x_ddf, color='orange', fill=False)\n",
+    "    ax.add_patch(wfd_region)\n",
+    "    ax.add_patch(ddf_region)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plot_ra_dec(df)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The overall object density distribution looks good.\n",
+    "\n",
+    "Notes:\n",
+    "* We're missing the DDF region, which was specifically not included in this DR1 coadd.\n",
+    "* We're missing a bit of the South-West (lower-left, \"North Up, East Left\") corner.\n",
+    "* There are also a few patches that failed within the main region.\n",
+    "* There is an overall gradient N/S in object density, because we're plotting in rectilinear RA, Dec bins, which means that bins at the bottom in RA cover less area than those at the top.\n",
+    "\n",
+    "See the input visit coverage map here:\n",
+    "https://github.com/LSSTDESC/ImageProcessingPipelines/issues/97#issuecomment-498303504\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Select good detections:\n",
+    "#  1. Marked as 'good' in catalog flags.\n",
+    "#  2. SNR in given band > threshold\n",
+    "#  3. In defined simulation range\n",
+    "snr_threshold = 5\n",
+    "snr_filter = 'i'\n",
+    "snr = df['psFlux_%s' % snr_filter] / df['psFluxErr_%s' % snr_filter]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def ellipticity(I_xx, I_xy, I_yy):\n",
+    "    \"\"\"Calculate ellipticity from second moments.\n",
+    "\n",
+    "    Parameters\n",
+    "    ----------\n",
+    "    I_xx : float or numpy.array\n",
+    "    I_xy : float or numpy.array\n",
+    "    I_yy : float or numpy.array\n",
+    "\n",
+    "    Returns\n",
+    "    -------\n",
+    "    e, e1, e2 : (float, float, float) or (numpy.array, numpy.array, numpy.array)\n",
+    "        Complex ellipticity, real component, imaginary component\n",
+    "        \n",
+    "    Copied from https://github.com/lsst/validate_drp/python/lsst/validate/drp/util.py\n",
+    "    \"\"\"\n",
+    "    e = (I_xx - I_yy + 2j*I_xy) / (I_xx + I_yy + 2*SM.sqrt(I_xx*I_yy - I_xy*2))\n",
+    "    e1 = np.real(e)\n",
+    "    e2 = np.imag(e)\n",
+    "    return e, e1, e2"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for filt in filters:\n",
+    "    df['e_{}'.format(filt)], df['e1_{}'.format(filt)], df['e2_{}'.format(filt)] = \\\n",
```

Comment:
When I get to this cell on jupyter.nersc.gov the kernel silently dies.  I think it's running out of memory.

So I can't really review beyond here.

Reply:
Can you try with the restricted one-tract catalog?
I've added it as an option in the first cell to allow someone to run the Notebook in a finite amount of time.

I will also add a completed run to the `rendered` branch.